sampler.go

```diff
@@ -497,7 +513,7 @@ func (s *RemotelyControlledSampler) extractSampler(
 		if sampler, ok := s.sampler.(*adaptiveSampler); ok {
 			return sampler, strategies, nil
 		}
-		sampler, err := NewAdaptiveSampler(strategies)
+		sampler, err := NewAdaptiveSampler(strategies, defaultMaxOperations)
```

Comment:
Sure makes sense, do we want it configurable via the sampler endpoint or per client? I think per client makes more sense